src/Amazon.Extensions.CognitoAuthentication/CognitoUserAuthentication.cs

```diff
@@ -63,14 +63,11 @@ public async Task<AuthFlowResponse> StartWithSrpAuthAsync(InitiateSrpAuthRequest
 
             UpdateSessionIfAuthenticationComplete(verifierResponse.ChallengeName, verifierResponse.AuthenticationResult);
 
-            return new AuthFlowResponse()
-            {
-                SessionID = verifierResponse.Session,
-                ChallengeName = verifierResponse.ChallengeName,
-                AuthenticationResult = verifierResponse.AuthenticationResult,
-                ChallengeParameters = verifierResponse.ChallengeParameters,
-                ClientMetadata = new Dictionary<string, string>(verifierResponse.ResponseMetadata.Metadata)
-            };
+            return new AuthFlowResponse(verifierResponse.Session,
```

Comment:
(minor) if you want to make this more readable, you can still use named parameters.